configs/sst_rhel_lightspeed.yaml

```diff
@@ -15,7 +15,19 @@ data:
           description: Command Line Assistant
           dependencies:
             - python3
-
+            - python3-dasbus
+            - python3-requests
+            - python3-sqlalchemy
+            - systemd
+    - srpm_name: command-line-assistant-selinux
+      limit_arches:
+        - aarch64
+        - x86_64
+      rpms:
```

Comment:
The -selinux subpackage is just that, a new RPM under the existing SRPM, so these lines are incorrect.
```suggestion
```